source/common/upstream/cluster_manager_impl.h

```diff
@@ -111,12 +111,15 @@ class ClusterManagerInitHelper : Logger::Loggable<Logger::Id::upstream> {
 
   enum class State {
     // Initial state. During this state all static clusters are loaded. Any primary clusters
-    // are immediately initialized.
+    // are immediately begin initialization.
```

Comment:
? I think what was there before made more sense?

Reply:
I was actually confused by this since 'immediately initialized` meant to me `immediately complete initialization` (and I have also assumed that initialized cluster have completed discovery of endpoints). Which is not the case for DNS based clusters these with health checks. I can revert this if I'm confusing the meaning of this statement.

Reply:
Ah OK, I think just a grammar issue (remove "are"):
"... immediately begin initialization"

Reply:
All your base are belong to us